app/helpers/measures_helper.rb

```diff
@@ -1,12 +1,41 @@
 module MeasuresHelper
-  def order_number_info(record)
-    record.definition.try(:description) || "Order number #{record.number}"
+  def show_warning?(order_number)
+    order_number.definition.description && order_number.number.start_with?('05')
+  end
+
+  def start_and_end_dates_for(definition)
+    start_date = definition.validity_start_date
+    end_date = definition.validity_end_date
+
+    "#{start_date.to_formatted_s(:gov)} to #{end_date.to_formatted_s(:gov)}"
+  end
+
+  def suspension_period_dates_for(definition)
+    if definition.suspension_period_start_date.present? && definition.suspension_period_end_date.present?
+      start_date = definition.suspension_period_start_date
+      end_date = definition.suspension_period_end_date
+
+      "#{start_date.to_formatted_s(:gov)} to #{end_date.to_formatted_s(:gov)}"
+    else
+      'n/a'
+    end
```

Comment:
Fancy using:

```ruby
 return 'n/a' unless foo

bar
```
?

Not a blocker.

Reply:
I did originally and found it less readable :thinking:

Reply:
I think rust and elixir have ruined me